core-bundle/contao/drivers/DC_Table.php

```diff
@@ -1130,7 +1130,7 @@ protected function copyChildren($table, $insertID, $id, $parentId)
 			$children = Input::get('childs');
 		}
 
-		if ($children && $db->fieldExists('pid', $table) && $db->fieldExists('sorting', $table))
+		if ($children && ($GLOBALS['TL_DCA'][$table]['list']['sorting']['mode'] ?? null) == self::MODE_TREE)
```

Comment:
```suggestion
		if (!($GLOBALS['TL_DCA'][$table]['config']['ptable'] ?? null) && $children && $db->fieldExists('pid', $table) && $db->fieldExists('sorting', $table))
```
This would be an alternative that just reverts the change from https://github.com/contao/contao/pull/6521#pullrequestreview-1777396112

Reply:
@Toflar Does this also fix the issue?

Reply:
Yes, that also works - but I am not using any nested elements just yet so I don't know how it affects those. But copying the entire page tree with children works with both fixes.

Reply:
I tested the new fix for copying nested elements and it worked :+1: